### DISCUSSION TITLE
Options to customize the profile (profile pictures, colours)

### DISCUSSION THREAD

**Discussion by Phyloiota** [iotaledger/firefly#1564] [Features / Enhancements]
## Description

Possibility to upload profile pictures and change color of the profile badge

## Motivation

user experience

## Requirements

add an option in the profile settings to change bagde color or replace standard badge with uploaded picture

1.
2.
3.

## Open questions (optional)

Use this section to ask any questions that are related to the feature.

## Are you planning to do it yourself in a pull request?

/No.

**Comment by cvarley100** [comment]
This will be added in a future version.
